### ISSUE TITLE
Problem With Flutter 2.8.0 [Gestures]

### ISSUE THREAD

**Issue by liemfs** [fluttercandies/extended_image#428]
Flutter version 2.8.0 has just been released. When I build the project using that version of Flutter, I get an error message from this library, saying that there are incorrect parameters.

### Flutter Doctor
```
Doctor summary (to see all details, run flutter doctor -v):
[√] Flutter (Channel stable, 2.8.0, on Microsoft Windows [Version 10.0.22000.348], locale id-ID)
[√] Android toolchain - develop for Android devices (Android SDK version 31.0.0)
[X] Chrome - develop for the web (Cannot find Chrome executable at .\Google\Chrome\Application\chrome.exe)
    ! Cannot find Chrome. Try setting CHROME_EXECUTABLE to a Chrome executable.
[√] Android Studio (version 2020.3)
[√] VS Code, 64-bit edition (version 1.62.3)
[√] Connected device (1 available)

! Doctor found issues in 1 category.
```
### Flutter Build Error
```
/C:/Users/nevergonnagiveyouup/Documents/Flutter%20SDK/.pub-cache/hosted/pub.dartlang.org/extended_image-5.1.3/lib/src/gesture_detector/scale.dart:32:11: Error: No named parameter with the name 'delta'.
          delta: delta,
          ^^^^^
/C:/Users/nevergonnagiveyouup/Documents/Flutter%20SDK/.pub-cache/hosted/pub.dartlang.org/extended_image-5.1.3/lib/src/gesture_detector/scale.dart:339:43: Error: Too few positional arguments: 2 required, 1 given.
          focalPointDelta > computePanSlop(pointerDeviceKind))
                                          ^
/C:/Users/nevergonnagiveyouup/Documents/Flutter%20SDK/packages/flutter/lib/src/gestures/events.dart:2116:8: Context: Found this candidate, but the arguments don't match.
double computePanSlop(PointerDeviceKind kind, DeviceGestureSettings? settings) {
       ^^^^^^^^^^^^^^
/C:/Users/nevergonnagiveyouup/Documents/Flutter%20SDK/.pub-cache/hosted/pub.dartlang.org/extended_image-5.1.3/lib/src/gesture_detector/drag.dart:46:66: Error: Too few positional arguments: 2 required, 1 given.
    final double minDistance = minFlingDistance ?? computeHitSlop(kind);
                                                                 ^
/C:/Users/nevergonnagiveyouup/Documents/Flutter%20SDK/packages/flutter/lib/src/gestures/events.dart:2103:8: Context: Found this candidate, but the arguments don't match.
double computeHitSlop(PointerDeviceKind kind, DeviceGestureSettings? settings) {
       ^^^^^^^^^^^^^^
/C:/Users/nevergonnagiveyouup/Documents/Flutter%20SDK/.pub-cache/hosted/pub.dartlang.org/extended_image-5.1.3/lib/src/gesture_detector/drag.dart:54:55: Error: Too few positional arguments: 2 required, 1 given.
    return _globalDistanceMoved.abs() > computeHitSlop(pointerDeviceKind);
                                                      ^
/C:/Users/nevergonnagiveyouup/Documents/Flutter%20SDK/packages/flutter/lib/src/gestures/events.dart:2103:8: Context: Found this candidate, but the arguments don't match.
double computeHitSlop(PointerDeviceKind kind, DeviceGestureSettings? settings) {
       ^^^^^^^^^^^^^^
/C:/Users/nevergonnagiveyouup/Documents/Flutter%20SDK/.pub-cache/hosted/pub.dartlang.org/extended_image-5.1.3/lib/src/gesture_detector/drag.dart:101:66: Error: Too few positional arguments: 2 required, 1 given.
    final double minDistance = minFlingDistance ?? computeHitSlop(kind);
                                                                 ^
/C:/Users/nevergonnagiveyouup/Documents/Flutter%20SDK/packages/flutter/lib/src/gestures/events.dart:2103:8: Context: Found this candidate, but the arguments don't match.
double computeHitSlop(PointerDeviceKind kind, DeviceGestureSettings? settings) {
       ^^^^^^^^^^^^^^
/C:/Users/nevergonnagiveyouup/Documents/Flutter%20SDK/.pub-cache/hosted/pub.dartlang.org/extended_image-5.1.3/lib/src/gesture_detector/drag.dart:109:55: Error: Too few positional arguments: 2 required, 1 given.
    return _globalDistanceMoved.abs() > computeHitSlop(pointerDeviceKind);
                                                      ^
/C:/Users/nevergonnagiveyouup/Documents/Flutter%20SDK/packages/flutter/lib/src/gestures/events.dart:2103:8: Context: Found this candidate, but the arguments don't match.
double computeHitSlop(PointerDeviceKind kind, DeviceGestureSettings? settings) {
       ^^^^^^^^^^^^^^


FAILURE: Build failed with an exception.

* Where:
Script 'C:\Users\nevergonnagiveyouup\Documents\Flutter SDK\packages\flutter_tools\gradle\flutter.gradle' line: 1070

* What went wrong:
Execution failed for task ':app:compileFlutterBuildRelease'.
> Process 'command 'C:\Users\nevergonnagiveyouup\Documents\Flutter SDK\bin\flutter.bat'' finished with non-zero exit value 1

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 2m 9s
```

**Comment by CoderJava** [comment]
Yes. I have the same issue.

**Comment by tcdspring** [comment]
I have the same issue too.

**Comment by EricPHPH** [comment]
same here

**Comment by shofizone** [comment]
same here

**Comment by mdddj** [comment]
同样的问题

**Comment by mdddj** [comment]
fix bugs with flutter 2.8.0
```dart
  extended_image:
    git:
      url: git://github.com/mdddj/extended_image.git
```

**Comment by oliverbuddy** [comment]
+1

**Comment by lazyee** [comment]
+1

**Comment by lazyee** [comment]
> fix bugs with flutter 2.8.0
> 
> ```dart
>   extended_image:
>     git:
>       url: git://github.com/mdddj/extended_image.git
> ```

如果用了其他依赖extended_image的库,例如wechat_assets_picker的话,会报来源不一致的问题,我还是蹲一波更新吧

**Comment by araisann** [comment]
> > fix bugs with flutter 2.8.0
> > ```dart
> >   extended_image:
> >     git:
> >       url: git://github.com/mdddj/extended_image.git
> > ```
> 
> 如果用了其他依赖extended_image的库,例如wechat_assets_picker的话,会报来源不一致的问题,我还是蹲一波更新吧

```
dependency_overrides:
  extended_image:
    git:
      url: git://github.com/mdddj/extended_image.git
```

**Comment by zmtzawqlp** [comment]
https://github.com/fluttercandies/extended_image/issues/400
